spec/support/curl_double.rb

```diff
@@ -0,0 +1,24 @@
+class CurlDouble
+  class Response
+    attr_accessor :body
+
+    def initialize(body)
+      @body = body
+    end
+  end
+
+  attr_accessor :stubbed_get_body, :stubbed_post_body
+
+  def initialize(get = "", post = "")
+    @stubbed_get_body = get
+    @stubbed_post_body = post
+  end
+
+  def get(url, params = {}, &block)
```

Comment:
Unused method argument - `url`. If it's necessary, use `_` or `_url` as an argument name to indicate that it won't be used. You can also write as `get(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `params`. If it's necessary, use `_` or `_params` as an argument name to indicate that it won't be used. You can also write as `get(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used. You can also write as `get(*)` if you want the method to accept any arguments but don't care about them.